yeat/tests/test_bandage.py

```diff
@@ -0,0 +1,35 @@
+# -------------------------------------------------------------------------------------------------
+# Copyright (c) 2022, DHS. This file is part of YEAT: http://github.com/bioforensics/yeat
+#
+# This software was prepared for the Department of Homeland Security (DHS) by the Battelle National
+# Biodefense Institute, LLC (BNBI) as part of contract HSHQDC-15-C-00064 to manage and operate the
+# National Biodefense Analysis and Countermeasures Center (NBACC), a Federally Funded Research and
+# Development Center.
+# -------------------------------------------------------------------------------------------------
+
+import pytest
+import subprocess
+from unittest.mock import patch
+from yeat.bandage import bandage
+
+
+def test_checking_bandage():
+    assert bandage.check_bandage()
```

Comment:
This test is simple but I thought it was worth some merit to include it. If the developer does not have Bandage in  their path, this assert statement will fail.

Even if this test was excluded, all of tests that run the entire YEAT workflow will fail because the workflow includes the Bandage workflow, which requires the $PATH to be updated.

Maybe I should create a --bandage-off flag to disable it?

Reply:
I guess it depends on whether you want to make Bandage mandatory for running the test suite. That would be the easiest approach from the developer standpoint. But it would cause the test suite to fail for users that can't or don't want to install Bandage. And since YEAT will work fine without Bandage, you may want to accommodate that.

Consider using `pytest.mark` to mark this test. You could then run something like `pytest -m bandage` or `pytest -m 'not bandage'` like we do with "drmaa" for Mystt. (Of course, you'd probably want a Makefile shortcut for that.) This will facilitate running the full test suite when Bandage is installed, but excluding this test when it is not.

Reply:
A few tests now have the `bandage` pytest.mark decorator. All tests but `make testbandage` will exclude bandage required tests.